### DISCUSSION TITLE
[Neo-tree WARN] Error rendering component icon

### DISCUSSION THREAD

**Discussion by hopezh** [nvim-neo-tree/neo-tree.nvim#1430] [Q&A] [answered]
I suddenly got the following warning every time I open neo-tree. May I ask what it means and how to fix it? Thanks. 

My neovim config is [here](https://github.com/hopezh/oatVim06). 

![2024-04-09 23 25 37](https://github.com/nvim-neo-tree/neo-tree.nvim/assets/23435810/3cd0025a-a8df-4c20-be91-ab1e7171944d)

**Comment by pysan3** [comment, marked as answer]
- https://github.com/nvim-tree/nvim-web-devicons/issues/440

It looks like an upstream issue ;)

**Comment by hopezh** [comment]
Ok, seems the issue is gone after updating my lazyvim config...

**Reply by pysan3** [reply]
Awesome. Thanks for reporting 👍

**Comment by rushin236** [comment]
Can some one help with this error I am not able to understand it

![image](https://github.com/nvim-neo-tree/neo-tree.nvim/assets/56204964/f7e9eeed-e8d1-44b1-ac86-7a3e1260e334)

Nvim Version = 0.10stable

Following is the neo tree config:
return {
  "nvim-neo-tree/neo-tree.nvim",
  branch = "v3.x",
  cmd = "Neotree",
  version = "*",
  dependencies = {
    "nvim-lua/plenary.nvim",
    "nvim-tree/nvim-web-devicons", -- not strictly required, but recommended
    "MunifTanjim/nui.nvim",
  },
  keys = {
    { "<leader>e", "<cmd>Neotree toggle<cr>", desc = "Toggle File Explorer" },
    { "<leader>be", "<cmd>Neotree float buffers<cr>", desc = "Toggle Buffer Explorer" },
    { "<leader>ge", "<cmd>Neotree float git_status<cr>", desc = "Toggle Git Status Explorer" },
  },
  config = function()
    -- If you want icons for diagnostic errors, you'll need to define them somewhere:
    vim.fn.sign_define("DiagnosticSignError", { text = " ", texthl = "DiagnosticSignError" })
    vim.fn.sign_define("DiagnosticSignWarn", { text = " ", texthl = "DiagnosticSignWarn" })
    vim.fn.sign_define("DiagnosticSignInfo", { text = " ", texthl = "DiagnosticSignInfo" })
    vim.fn.sign_define("DiagnosticSignHint", { text = "󰌵", texthl = "DiagnosticSignHint" })

    local neo_tree = require("neo-tree")

    neo_tree.setup({
      sources = { "filesystem", "buffers", "git_status", "document_symbols" },
      open_files_do_not_replace_types = { "terminal", "Trouble", "trouble", "qf", "Outline" },
      filesystem = {
        filesystem = {
          filtered_items = {
            visible = true,
            hide_dotfiles = false,
            hide_hidden = false,
            hide_gitignore = false,
          },
        },
        bind_to_cwd = false,
        follow_current_file = { enabled = true },
        use_libuv_file_watcher = true,
      },
      window = {
        mappings = {
          ["<space>"] = "none",
          ["Y"] = {
            function(state)
              local node = state.tree:get_node()
              local path = node:get_id()
              vim.fn.setreg("+", path, "c")
            end,
            desc = "copy path to clipboard",
          },
        },
      },
      default_component_configs = {
        indent = {
          with_expanders = true, -- if nil and file nesting is enabled, will enable expanders
          expander_collapsed = "",
          expander_expanded = "",
          expander_highlight = "NeoTreeExpander",
        },
      },
    })
  end,
}

**Reply by cseickel** [reply]
This is completely unrelated to this discussion. You should start a new one rather than hijacking this one for something unrelated.
